.cargo/config.toml

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: © 2023 Rivos Inc.
```

Comment:
Apparently the Unicode character for Copyright is not needed if SPDX-FileCopyrightText is used. Can we get rid of Copyright symbol? Might cause problem with some editors/terminal combinations.

Reply:
done, replaced with ascii version